packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -338,3 +339,32 @@ export function updateOutputPort(node: WorkflowNode<any>, updatedOutputPort: Wor
 		...updatedOutputPort
 	};
 }
+
+export const getGroupedOutputs = <T>(
+	node: WorkflowNode<any>,
+	labels: { saved?: string; unsaved?: string }
+) => {
```

Comment:
Can you provide some comments/description on what this function is doing? 

It feels very odd both in terms of the function-signature, and as well the logic itself.

Reply:
added!